src/main/webapp/app/shared/util/firebase/firebase-utils.tsx

```diff
@@ -500,7 +527,48 @@ export function compareMutationsByCategoricalAlteration(mut1: Mutation, mut2: Mu
   }
 }
 
-export function compareMutations(mut1: Mutation, mut2: Mutation) {
+export function getMutationModifiedTimestamp(mutation: Mutation): number | null {
+  let modifiedTime: number = null;
+  let nestedObjects = [mutation];
+  while (nestedObjects.length > 0) {
+    const newNestedObjects = [];
+
+    for (const nestedObject of nestedObjects) {
+      for (const [key, val] of Object.entries(nestedObject)) {
+        if (key.endsWith('review')) {
```

Comment:
on the safe side, I would do 
```suggestion
        if (key.endsWith('_review')) {
```